extension.html

```diff
@@ -0,0 +1,119 @@
+      var respecConfig = {
+        specStatus: "CG-DRAFT",
+        edDraftURI: "https://w3c.github.io/pointerevents/extensions.html",
+        editors: [{
+          name:        "Navid Zolghadr",
+          url:         "https://github.com/NavidZ",
+          company:     "Google",
+          companyURL:  "https://www.google.com/"
+        }],
+        shortName: "PointerEvent Level 2 Extensions",
+        otherLinks: [{
+          key: 'Participation',
+          data: [
+              {
+                value: 'GitHub repository',
+                href: 'https://github.com/w3c/pointerevents/'
+              } ,{
+                value: 'File a bug / view open issues',
+                href: 'https://github.com/w3c/pointerevents/issues'
+              }
+            ]
+          }],
+      };
+    </script>
+  </head>
+  <body>
+    <section id='abstract'>
+      <p>
+The features in this specification extend or modify those found in <a href="https://w3c.github.io/pointerevents/index.html">Pointer Events Level 2</a> that describes events and related interfaces for handling hardware agnostic pointer input from devices including a mouse, pen, touchscreen, etc.
+      </p>
+    </section>
+
+    <section id='sotd'>
+      <p>
+        This proposal was first introduced in this <a href="https://discourse.wicg.io/t/new-event-for-non-primary-button-click/1527">discourse</a>.
+      </p>
+    </section>
+
+    <section id='intro' class='informative'>
+      <h2>Introduction</h2>
+      <p>
+        This document mainly contains the new APIs that may not make it to Pointer Event Level 2 spec. So we iterate over these new APIs to make them ready for the next version.
+      </p>
+      <p>
+	One of the APIs that are added in this document exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps might want to take advantage of the a precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. So we needed an API to expose the coalesced events along with the event that was dispatched to the javascript. 
+      </p>
+    </section>
+    <section dfn-for="extensions-to-pointerevents-interface">
+      <h2>Extensions to the <code>PointerEvent</code> interface</h2>
+      <p>
+        The following section describes extensions to the existing <a href="https://w3c.github.io/pointerevents/index.html#pointerevent-interface"><code>PointerEvent</code> interface</a>.
+      </p>
+      <div>
+          <pre class="idl">
+partial dictionary PointerEventInit {
+    sequence&lt;PointerEventInit> coalescedEventInits = [];
```

Comment:
I can see how this coalescedEventInits pattern would be convenient for developers writing events from scratch, but I don't think it's worth worrying about that too much (it's a very niche scenario).  I think it would be unusual for a constructor to involve logic like this, normally they just populate properties.  So I suggest we change this to instead be `sequence<PointerEvent> coalescedEvents = [];`